docker-compose.yml

```diff
@@ -2,21 +2,20 @@
 # For production we recommend to rather use
 # image: kartoza/pg-backup
 # in the dbbackup service
-db:
-  image: kartoza/postgis
-  environment:
-    - USERNAME=docker
-    - PASS=docker
-
-dbbackup:
+sftpbackup:
   build: .
   volumes:
     - ./backup:/backup
+    - ./target:/target
   environment:
-    - DUMPPREFIX=watchkeeper
-    - PGHOST=db
-    - PGUSER=docker
-    - PGPASSWORD=docker
-    - PGPORT=5432
-  links:
-    - db:db
+    - DUMPPREFIX=backup
```

Comment:
yeah great thanks! Perhaps we could put in small comment lines above each option explaining what the numbers mean as it is not immediately obvious what they do.